Apps/Admin/Client/Program.cs

```diff
@@ -99,6 +100,8 @@ public static async Task Main(string[] args)
 
             builder.Services.AddBlazoredLocalStorage();
 
+            builder.Services.AddTransient<IDateConversionService, DateConversionService>();
```

Comment:
```suggestion
            builder.Services.AddSingleton<IDateConversionService, DateConversionService>();
```
The same conversion service should be able to be used throughout the lifetime of the application as it only stores data related to the Configuration.